client/src/main/java/org/apache/rocketmq/client/impl/factory/MQClientInstance.java

```diff
@@ -255,19 +255,17 @@ public void start() throws MQClientException {
     }
 
     private void startScheduledTask() {
-        if (null == this.clientConfig.getNamesrvAddr()) {
```

Comment:
@lizhanhui @zhouxinyu  @shroman @Jaskey 
we can only keep one between two of them, am i right?

if (null == this.clientConfig.getNamesrvAddr()) { this.mQClientAPIImpl.fetchNameServerAddr(); }

and

private void startScheduledTask() {
if (null == this.clientConfig.getNamesrvAddr()) {.......}